waspc/src/Wasp/Generator/WebAppGenerator.hs

```diff
@@ -113,7 +126,8 @@ npmDepsForWasp spec =
             ("react-dom", "^17.0.2"),
             ("@tanstack/react-query", "^4.13.0"),
             ("react-router-dom", "^5.3.3"),
-            ("react-scripts", "5.0.1")
+            ("react-scripts", "5.0.1"),
+            ("@prisma/client", show prismaVersion)
```

Comment:
The web app only needs `@prisma/client` (we're using the server's CLI to generate what's necessary.